apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -36,7 +36,19 @@ const generateHashedLink = (id: number) => {
   return uid;
 };
 
-export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupInfered, "eventType" | "team">) => {
+const getRandomId = (length = 8) => {
```

Comment:
customInput type requires the id to be of type number and within 32 bit integer so i created this random id generator function. This is the temporary id we'll be using until user clicks save and server save these custom inputs and return correct id.  

our current approach of using id as -1 was causing this bug https://photos.google.com/share/AF1QipMCbabiovDIRlru0ASfB0uerBr-aoN4XLG0_xTwAerAzAm1FFeRA4Kp-yf-Yi4bWA/photo/AF1QipP42jgzOHOeraQ45gF0MFcHCRxcEhjfl1dfyMq1?key=ZkZ0ZnZhQ2d0UXNYdE8wSHFmaWJjNTdsaWdGRmp3